COLLADA2GLTFWriter.cpp

```diff
@@ -319,6 +319,14 @@ namespace GLTF
                 (*materialBindingsPrimitiveMap)[primitive->getMaterialObjectID()] = materialBinding;
             }
         }
+
+        // This optimization will prevent incredibly inefficient O(n^2) scanning in writeNode()
```

Comment:
This comment won't make sense once this is merged since the code it refers to won't exist anymore; probably best to remove it.